packages/clay-form/src/DualListBox.tsx

```diff
@@ -56,8 +56,8 @@ interface IProps extends React.HTMLAttributes<HTMLDivElement> {
 	 * Labels for the aria attributes
 	 */
 	ariaLabels?: {
-		transferButtonLTR: string;
-		transferButtonRTL: string;
+		transferLTR: string;
```

Comment:
For all ariaLabel props, I think we should focus less on the element itself and more on the language the user wants to use. So by removing `Button` from the name, it's more clear that this prop is for a language map.

Reply:
Yea the naming was a bit tricky to decide on